### PR TITLE
Feature/253 remaining password property rules

### PR DESCRIPTION
**Pull request by phelewski**
Partial for #253 

Creates rule for the following resource properties:
* AWS::CodePipeline::Webhook.WebhookAuthConfiguration SecretToken
* AWS::ElastiCache::ReplicationGroup AuthToken
* AWS::Lambda::Permission EventSourceToken
* AWS::ManagedBlockchain::Member.MemberFabricConfiguration AdminPassword
* AWS::DocDB::DBCluster MasterUserPassword

### CODE REVIEW

File: lib/cfn-nag/custom_rules/ManagedBlockchainMemberMemberFabricConfigurationAdminPasswordRule.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  end
+
+  def rule_id
+    'F71'
+  end
+
+  def audit_impl(cfn_model)
+    managed_blockchain_members = cfn_model.resources_by_type('AWS::ManagedBlockchain::Member')
+    violating_managed_blockchains = managed_blockchain_members.select do |member|
+      member_config = member.memberConfiguration['MemberFrameworkConfiguration']['MemberFabricConfiguration']['AdminPassword']
```
Comment (erickascic) [inline lib/cfn-nag/custom_rules/ManagedBlockchainMemberMemberFabricConfigurationAdminPasswordRule.rb]: MemberFabricConfiguration is optional

Comment (erickascic) [reply lib/cfn-nag/custom_rules/ManagedBlockchainMemberMemberFabricConfigurationAdminPasswordRule.rb]: which looks like you got, but don't want to dereference the AdminPassword then?

Comment (phelewski) [reply lib/cfn-nag/custom_rules/ManagedBlockchainMemberMemberFabricConfigurationAdminPasswordRule.rb]: I've added some logic here to ensure all of these optional properties exist.

File: lib/cfn-nag/custom_rules/ManagedBlockchainMemberMemberFabricConfigurationAdminPasswordRule.rb [hunk 2]
```diff
@@ -22,15 +22,34 @@ def rule_id
...
     managed_blockchain_members = cfn_model.resources_by_type('AWS::ManagedBlockchain::Member')
     violating_managed_blockchains = managed_blockchain_members.select do |member|
-      member_config = member.memberConfiguration['MemberFrameworkConfiguration']['MemberFabricConfiguration']['AdminPassword']
-      if member_config.nil?
+      if password_property_does_not_exist(member)
         false
       else
-        insecure_parameter?(cfn_model, member_config) ||
-          insecure_string_or_dynamic_reference?(cfn_model, member_config)
+        pw = member.memberConfiguration['MemberFrameworkConfiguration']['MemberFabricConfiguration']['AdminPassword']
```
Comment (erickascic) [inline lib/cfn-nag/custom_rules/ManagedBlockchainMemberMemberFabricConfigurationAdminPasswordRule.rb]: if MemberFabricConfiguration isn't specifed you'll get an NPE here?

**Pull request merged**